name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/LGTBankPDFExtractor.java

```diff
@@ -55,22 +65,19 @@ private void addBuySellTransaction()
             // ISIN DK0010244508
             // Valorennummer 906020
             // Wertpapierkennnummer 861837
-            .section("isin", "wkn", "name")
+            .section("isin", "wkn", "name", "nameContinued")
             .match("^(Titel) (?<name>.*)$")
+            .match("(?<nameContinued>.*)")
             .match("^(ISIN) (?<isin>[\\w]{12}.*)$")
-            .match(".*")
             .match("^(Wertpapierkennnummer) (?<wkn>.*)$")
             .assign((t, v) -> {
                 t.setSecurity(getOrCreateSecurity(v));
             })
 
-            // Abschlussdatum 14.04.2020 09:00:02
-            .section("date", "time")
-            .match("^(Abschlussdatum) (?<date>\\d+.\\d+.\\d{4}+) (?<time>\\d+:\\d+:\\d+)$")
+            // Valuta 16.04.2020
+            .section("date")
+            .match("^(Valuta) (?<date>\\d+.\\d+.\\d{4}+)$")
```

Comment:
Ich schlage vor für Kaufbuchungen werden wir *nicht* das Valuta Datum sondern das Abschussdatum. An diesem Tag (und Uhrzeit) hat man das Wertpapier gekauft. Ich finde das reflektiert die Wirklichkeit besser als das die Bank noch 2 Tage braucht um Dir das Wertpapier ins Depot zu schieben.


Ich weiß bei Dividenden kann man anders argumentieren (und da gibt es ja immer die Nachfrage/Idee tatsächliche zwei Datumsfelder in der Buchung zu speichern).

Reply:
Okay :-)